sdk/include/psdk/ntgdi.h

```diff
@@ -1663,11 +1663,11 @@ BOOL
 APIENTRY
 NtGdiGetTextExtentExW(
     _In_ HDC hdc,
-    _In_reads_opt_(cwc) LPWSTR pwsz,
+    _In_reads_opt_(cwc) LPCWSTR pwsz,
```

Comment:
```suggestion
    _In_reads_opt_(cwc) PCWSTR pwsz,
```
or `PCWCH` if it's a non-NUL-terminated string.

Reply:
```suggestion
    _In_reads_opt_(cwc) PCWCH pwsz,
```